plugin/core/windows.py

```diff
@@ -393,15 +393,19 @@ def on_post_exit_async(self, session: Session, exit_code: int, exception: Option
             listener.on_session_shutdown_async(session)
         if exit_code != 0 or exception:
             config = session.config
-            msg = "".join((
-                "{0} exited with status code {1}. ",
-                "Do you want to restart it? If you choose Cancel, it will be disabled for this window for the ",
-                "duration of the current session. ",
-                "Re-enable by running \"LSP: Enable Language Server In Project\" from the Command Palette."
-            )).format(config.name, exit_code)
-            if exception:
-                msg += "\n\n--- Error: ---\n{}".format(str(exception))
-            if sublime.ok_cancel_dialog(msg, "Restart {}".format(config.name)):
+            restart = self._config_manager.record_crash(config.name, exit_code, exception)
+            if not restart:
+                msg = "".join((
+                    "The {0} server has crashed {1} times in the last {2} seconds.\n",
+                    "Exit code: {3}\n\n",
```

Comment:
I've removed this on purpose because:
 - it's quite low-level
 - it will be logged int he console anyway

Reply:
I had some issues in the past where users just posted the message or a screenshot. In these cases, the exit code and exception were quite useful.

If we remove the exit code, should we remove the exception too?

Reply:
In my experience exit code in most cases doesn't provide actionable information as most users don't know what each codes mean for specific servers. The exception can be very useful for diagnosing the issue on the other hand. And for those that really want to look deep into it, they can find exit code in the console.

Reply:
Right. Updated the message :)